.eslintrc.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Sorry, somehow I ended up in another branch (likely to isolate the early selenium 3.x work).  I will merge in those changes, which include migrating to the standard ESLint config.